### PR TITLE
batch send apply_proposals to apply worker

### PR DESCRIPTION
**Pull request by javaforfun**

### CODE REVIEW

File: src/raftstore/store/worker/apply.rs [hunk 1]
```diff
@@ -1141,12 +1151,8 @@ impl Task {
         Task::Registration(Registration::new(peer))
     }
 
-    pub fn proposals(id: u64, region_id: u64, props: Vec<Proposal>) -> Task {
-        Task::Proposals(Proposals {
-            id: id,
-            region_id: region_id,
-            props: props,
-        })
+    pub fn proposals(proposals: Vec<RegionProposal>) -> Task {
```
Comment (BusyJay) [inline src/raftstore/store/worker/apply.rs]: The method is redundant.

**Comment by siddontang** [comment]
any benchmark?

File: src/raftstore/store/peer.rs [hunk 2]
```diff
@@ -642,14 +642,14 @@ impl Peer {
         self.get_store().applied_index_term == self.term()
     }
 
-    pub fn schedule_apply_proposals(&mut self, worker: &Worker<ApplyTask>) {
+    pub fn schedule_apply_proposals(&mut self) -> Option<RegionProposal> {
```
Comment (siddontang) [inline src/raftstore/store/peer.rs]: The `schedule` in the function name is not proper here now.

**Comment by siddontang** [comment]
LGTM

PTAL @BusyJay

**Comment by siddontang** [comment]
PTAL @BusyJay

**Pull request merged**